src/pages/components/tile/usage.mdx

```diff
@@ -387,6 +407,11 @@ are written out as a formula of width to height, such as 3:2. It’s important t
 remember that while two images can have the same aspect ratio, they can have
 varying images sizes.
 
+Experimental tiles are now available. This addition enhances accessibility and
+only applies to the visual, not the functions of tiles. Though we are not
+deprecating the current tiles, we encourage all design teams to use the new
+experimental tiles in their products moving forward.
+
```

Comment:
is ti correct for this text to be in an InlineNotification up top and here as well? 
<img width="811" alt="image" src="https://user-images.githubusercontent.com/40550942/214891390-6ecae5b6-e3a0-4210-94ad-cb142635930e.png">
<img width="813" alt="image" src="https://user-images.githubusercontent.com/40550942/214891487-7d29b309-f708-4393-bef3-28ac1bfe17e3.png">